app/controllers/site_controller.rb

```diff
@@ -1,9 +1,30 @@
 # frozen_string_literal: true
 
 class SiteController < ApplicationController
+  before_action :check_active_notifications
   def index
     @todays_events = Event.today
     @upcoming_events = Event.front_page_events(11)
     @banner_event = @upcoming_events.shift
   end
+
+  def check_active_notifications
+    # from an dto are integers of date_type
+    # some supported (most useful) types:
+    #   %m - Month, e.g. 8 for August, 1 for January
+    #   %d - Day in month
+    #   %H - Hour of day
+    flash_in_date_range = lambda { |date_from, date_to, date_type, msg|
+      time_now = Time.now.strftime(date_type).to_i
+      return unless time_now.between?(date_from, date_to)
+      flash[:notice] = view_context.sanitize(msg)
+    }
+
+    register_membership =
```

Comment:
I imagine that you split up the string to make parts of it reusable elsewhere. Wouldnt it then make sense to omit the space included and adding it here instead?